eng/scripts/GenerateReferenceAssemblies.ps1

```diff
@@ -1,10 +1,26 @@
 param(
     [switch]$ci
 )
-$ErrorActionPreference = 'stop'
 
+$ErrorActionPreference = 'stop'
 $repoRoot = Resolve-Path "$PSScriptRoot/../.."
 
-& "$repoRoot\eng\common\msbuild.ps1" -ci:$ci "$repoRoot/eng/CodeGen.proj" `
+try {
+  # eng/common/msbuild.ps1 builds the Debug configuration unless there's a $Configuration variable. Match that here.
+  & "$repoRoot\build.ps1"  -ci:$ci -nobl -buildNative -configuration Debug
```

Comment:
Building native assets to avoid warnings when running the `GenerateReferenceSources` target.

Reply:
How long does it take to build native assets?

Reply:
Looking at hhttps://dev.azure.com/dnceng/internal/_build/results?buildId=662159&view=logs&j=1b89928a-2219-5ef9-602f-f95beb3da4dc&t=bcce746a-741e-59b7-adec-a4a82d8c5e52&l=313 for example, 7 minutes and 40 seconds. I believe this aligns with a speedup in the next `MSBuild` invocation.

The overall build time goes down across all jobs comparing builds of this PR (2h 3m 45s average of 5 builds) with 'master' (2h 30m 45s average of 17 builds in the last 7 days). Performance wasn't the goal but it still seemed to improve. That said, 5 builds isn't that many.

Part of the improvement may come from only needing to build native assets once per build job. `-noBuildNative` was not possible everywhere before because the native assets were built only for a single architecture.